src/meta_schedule/schedule_rule/multi_level_tiling_tensor_core.cc

```diff
@@ -556,6 +556,11 @@ ScheduleRule ScheduleRule::MultiLevelTilingTensorCore(
     Optional<Integer> max_innermost_factor, Optional<Array<Integer>> vector_load_lens,
     Optional<Map<String, ObjectRef>> reuse_read, Optional<Map<String, ObjectRef>> reuse_write,
     bool use_software_pipeline) {
+  if (tile_binds.defined()) {
+    for (const String& tile_bind : tile_binds.value()) {
+      CHECK_NE(tile_bind, "threadIdx.x") << "Cannot bind to threadIdx.x when using tensor core.";
+    }
+  }
```

Comment:
Does this check (along with existing tests) now cover the regression this fixes? If not adding a test to protect against the regression in the future will surely save you and others time!

Reply:
Oops looks like @junrushao already asked my question, I trust the your judgements :)

Reply:
Yes this check covered the broken case that incorrectly using `threadIdx.x`. As a follow up, I'm thinking also adding some e2e search space generation test for the default config for each target.